aws/src/integration/java/org/apache/iceberg/aws/s3/TestS3FileIOIntegration.java

```diff
@@ -106,6 +132,33 @@ public void testNewInputStream() throws Exception {
     validateRead(s3FileIO);
   }
 
+  @Test
+  public void testNewInputStreamWithAccessPoint() throws Exception {
+    s3.putObject(PutObjectRequest.builder().bucket(bucketName).key(objectKey).build(),
+        RequestBody.fromBytes(contentBytes));
+    S3FileIO s3FileIO = new S3FileIO(clientFactory::s3);
+    s3FileIO.initialize(ImmutableMap.of(AwsProperties.S3_ACCESS_POINTS_PREFIX + bucketName,
+        testAccessPointARN(AwsIntegTestUtil.testRegion(), accessPointName)));
+    validateRead(s3FileIO);
+  }
+
+  @Test
+  public void testNewInputStreamWithCrossRegionAccessPoint() throws Exception {
```

Comment:
Just a side note that in https://github.com/aws/aws-sdk-java-v2/commit/51632cb58491e13f1d88e636353d4ce87b34e81d, a new config `multiRegionEnabled` is added, which defaults to true and controls cross-region access when using MRAP. The default works for us, so we probably don't need to add it for now.